Tools/CISupport/ews-build/factories.py

```diff
@@ -350,6 +350,9 @@ def __init__(self, platform, **kwargs):
         self.addStep(CompileWebKit(skipUpload=True))
         self.addStep(KillOldProcesses())
 
+        self.addStep(ValidateChange(verifyMergeQueue=True, verifyNoDraftForMergeQueue=True))
+        self.addStep(RunWebKitTests())
```

Comment:
Before we do this, I think we need what `CommitQueue` does where we check if we've already passed layout tests in EWS, right?

Reply:
Done in updated commit. Although it needs Django changes to be deployed first (e.g.: https://github.com/WebKit/WebKit/pull/2178, https://github.com/WebKit/WebKit/pull/2325 etc).